### PR TITLE
[SPARK-26426][SQL]fix ExpresionInfo assert error in windows operation system.

### PR DESCRIPTION
**Pull request by yanlin-Lynn**
## What changes were proposed in this pull request?
fix ExpresionInfo assert error in windows operation system, when running unit tests.

## How was this patch tested?
unit tests

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
ok to test

File: sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java [hunk 1]
```diff
@@ -79,7 +79,8 @@ public ExpressionInfo(
         assert name != null;
         assert arguments != null;
         assert examples != null;
-        assert examples.isEmpty() || examples.startsWith("\n    Examples:");
+        assert examples.isEmpty() || examples.startsWith("\n    Examples:")
```
Comment (HyukjinKwon) [inline sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java]: Let's use `System.lineSeparator`

**Review by HyukjinKwon — APPROVED** [review]
LGTM

File: sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java (hunk 1 above)
Comment (yanlin-Lynn) [reply sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java]: OK, this is better.

**Comment by HyukjinKwon** [comment]
Merged to master and branch-2.4.

**Pull request closed without merging**